### PR TITLE
171 dataerror value too long for type character varying50

### PR DESCRIPTION
**Pull request by PouryaMansouri**

### CODE REVIEW

File: nakhll_market/models.py [hunk 1]
```diff
@@ -1319,7 +1319,10 @@ class Tag(models.Model):
     """
     #TODO: Document
     """
-    name = models.CharField(max_length=50, verbose_name=_('نام تگ'))
+    name = models.CharField(max_length=127, verbose_name=_('نام تگ'),
+    error_messages={
+        'max_length': _('نام تگ باید کمتر از 127 کاراکتر باشد.'),}
```
Comment (likecodingloveproblems) [inline nakhll_market/models.py:1324]: Does serializer run model.full_clean method?

**Comment by likecodingloveproblems** [comment]
We must put validation for each field separately or does Django validate them automatically?
@PouryaMansouri @mohsenone

File: nakhll_market/models.py (hunk 1 above)
Comment (PouryaMansouri) [reply nakhll_market/models.py:1324]: NO, 
I check it and you get this error in response, so I check it in in serializer.
I think it is better to check in front too. 
any front validation helps us that we have fewer requests to the server.

On the other hand, Do you think 50 or 70 characters are enough for the title of the tag?

Comment (PouryaMansouri) [reply nakhll_market/models.py:1324]: NO, 
I check it and I don't get this error in response, so I check it in the serializer.
I think it is better to check in front too. 
any front validation helps us that we have fewer requests to the server.

On the other hand, Do you think 50 or 70 characters are enough for the title of the tag?

@likecodingloveproblems

**Pull request merged**